packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -1443,7 +1470,7 @@ void wasm_sapi_module_shutdown()
 #endif
 	if (php_wasm_sapi_module.ini_entries)
 	{
-		free(php_wasm_sapi_module.ini_entries);
+		free((void *)php_wasm_sapi_module.ini_entries);
```

Comment:
Why is that cast necessary?

Reply:
There was a warning when compiling. Because `php_wasm_sapi_module.ini_entries` is a const *char. Do you want me to remove this?

Reply:
Hm, if that's const then maybe it doesn't have to be freed at all 🤔 Yeah, let's remove it. It could be a follow-up PR – let's not block this one on a minor detail.